docs/docs/using-client-side-only-libraries.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using client side only packages
```

Comment:
```suggestion
title: Using Client-Side Only Packages
```
Using title case to match the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/).